### PR TITLE
Display a physical plan with IndexMergePath using the Explain [format="dot"] command

### PR DESCRIPTION
**Pull request by hailanwhu**
### What problem does this PR solve? 
(1) After a physical plan with IndexMergePath generated, it's time to use `explain` command to display the plan.
(2) Fix bug in `explain format="dot"`. 

Tests 
 - Unit test

### CODE REVIEW

File: planner/core/common_plans.go [hunk 1]
```diff
@@ -784,8 +784,11 @@ func (e *Explain) prepareTaskDot(p PhysicalPlan, taskTp string, buffer *bytes.Bu
 	buffer.WriteString(fmt.Sprintf("label = \"%s\"\n", taskTp))
 
 	if len(p.Children()) == 0 {
-		buffer.WriteString(fmt.Sprintf("\"%s\"\n}\n", p.ExplainID()))
-		return
+		if taskTp == "cop" {
+			buffer.WriteString(fmt.Sprintf("\"%s\"\n}\n", p.ExplainID()))
```
Comment (zz-jason) [inline planner/core/common_plans.go]: how about:
```go
fmt.Fprintf(buf, "\"%s\"\n}\n", p.ExplainID())
```

File: planner/core/common_plans.go [hunk 2]
```diff
@@ -784,8 +784,11 @@ func (e *Explain) prepareTaskDot(p PhysicalPlan, taskTp string, buffer *bytes.Bu
...
+			return
+		}
+		buffer.WriteString(fmt.Sprintf("\"%s\"\n", p.ExplainID()))
```
Comment (zz-jason) [inline planner/core/common_plans.go]: ditto

File: planner/core/common_plans.go (hunk 1 above)
Comment (hailanwhu) [reply planner/core/common_plans.go]: That is better, more clear.

**Comment by AilinKid** [comment]
Hah, It's better to extract `Fix bug in explain format="dot".` logic code to a new PR which is good for attaching label to it and tracing.

**Comment by hailanwhu** [comment]
Yes, I propose a new PR.

**Pull request closed without merging**